base/applications/rapps/include/rosui.h

```diff
@@ -116,46 +120,37 @@ class CPointerArray
         return DPA_Sort(m_hDpa, s_OnCompareItems, (LPARAM)this);
     }
 
-    INT Search(T* item, INT iStart, UINT uFlags)
+    INT Search(T* item, INT iStart = 0, UINT uFlags = 0)
     {
-        return DPA_Search(m_hDpa, item, 0, s_OnCompareItems, (LPARAM)this, 0);
+        return DPA_Search(m_hDpa, item, iStart, s_OnCompareItems, (LPARAM)this, uFlags);
     }
 };
 
-class CUiRect
-    : public RECT
-{
-public:
-    CUiRect()
-    {
-        left = right = top = bottom = 0;
-    }
 
-    CUiRect(INT l, INT t, INT r, INT b)
-    {
-        left = l;
-        right = r;
-        top = t;
-        bottom = b;
-    }
-};
+//
+// Units/Ways of UI measure
+//
 
 class CUiMargin
-    : public CUiRect
+    : public RECT // CRect
```

Comment:
`CUiMargin` now directly inherits from the basic `RECT` structure. I have kept (commented) the code that would be needed if instead we make it derive from ATL's `CRect` class.
Suggestions are welcome.